src/main/java/essenmakanan/command/AddIngredientCommand.java

```diff
@@ -27,7 +28,14 @@ public void executeCommand() {
             Ingredient newIngredient;
             try {
                 newIngredient = IngredientParser.parseIngredient(ingredient);
-                ingredients.addIngredient(newIngredient);
+                if (this.ingredients.exist(newIngredient.getName())) {
+                    // if ingredient already exists, update the quantity
+                    this.ingredients.updateIngredient(newIngredient);
```

Comment:
I like that you update the ingredient if the ingredient has already existed in the list.